lib/l10n/intl_en.arb

```diff
@@ -1051,14 +1051,24 @@
       "nbMembers": {}
     }
   },
-  "shared_space_member_could_not_be_added": "Shared Space Member could not be added",
+  "shared_space_member_could_not_be_added": "Could not be added",
   "@shared_space_member_could_not_be_added": {
     "type": "text",
     "placeholders": {}
   },
-  "shared_space_member_has_been_added": "Shared Space Member has been successfully added",
+  "shared_space_member_has_been_added": "Added successfully",
```

Comment:
it is based on design?

Reply:
No, the design does not have these messages

Reply:
please contact to Phuong to get correctly message.